lib/index.js

```diff
@@ -8,6 +9,7 @@ import { Test, Suite } from './Data.js'
 export default {
   QUnitAdapter: QUnitAdapter,
   JasmineAdapter: JasmineAdapter,
+  MochaAdapter: MochaAdapter,
```

Comment:
Ok. May I move this file to the `root` of the project ? I think there should be the `entry point`.

Maybe also doing this in another issue, where we do more small fixes ?